docs/azure_jumpstart_ag/contoso_supermarket/adx_dashboards/_index.md

```diff
@@ -0,0 +1,124 @@
+
+- On the Client VM, locate Data Emulator icon on the desktop.
+
+  ![Locate Data Emulator on the desktop](./img/locate_data_emulator_desktop.png)
+
+- Double click on the Data Emulator desktop icon to launch executable and generate sample data. Confirm by entering **Yes** or **Y** to start generating data, entering No or N will exit the tool. This tool generates data for the last 30 days. Say No or N to prevent regenerating sample data if it is generated earlier.
+
+  > **Note**: You can still generate additional sample data by running this tool multiple times, but there might be duplicate key errors and fails to generate data in subsequent attempts.
+
+  ![Confirm sample data generation](./img/confirm_sample_data_generation.png)
+
+  ![Generating sample data](./img/sample_data_generation.png)
+
+- From ADX open PoS Orders report to view simulated orders data. Allow some time to propagate data into the ADX database using an integrated data pipeline.
+
+  ![PoS Orders with simulated data](./img/adx_posorders_with_simulated_data.png)
+
+- PoS Orders dashboard report is configured to display data from the _"Last 1 hour"_ by default. To view all the simulated orders data, change report time range to _"Last 30 days"_ as shown in the picture below. Dashboard report will refresh data and display reports for the selected time range.
+
+  ![PoS Orders select time range](./img/adx_orders_report_select_timerange.png)
+
+  ![PoS Orders with simulated data for selected time range](./img/adx_posorders_with_simulated_data_selected_timerange.png)
+
+### Generate orders from Contoso Supermarket store applications
+
+- On the Agora client VM, open Edge browser. From the favorites bar review bookmarks created for PoS applications for different stores and environments.
+
+  ![PoS app bookmarks](./img/pos_app_edge_bookmarks.png)
+
+- From the bookmarks expand POS -> Chicago and select "POS Chicago - Customer".
+
+  ![Select POS Chicago application](./img/pos_app_edge_select_pos_chicago_customer.png)
+
+- Randomly add a few items to the cart.
+
+  ![Select PoS Chicago application](./img/chicago_pos_app_customer.png)
+
+- Click on Cart, review items, and click Place Order.
+
+  ![Select PoS Chicago application](./img/pos_chicago_customer_place_order.png)
+
+- Place additional orders from the same store by repeating the above steps. Try adding random items to each order to simulate orders from different customers of the store.
+
+- To view the orders placed form the Contoso Chicago store, go to ADX portal, under Dashboards open PoS Orders report and chose time range as _"Last 30 minutes"_.
+
+  > **Note:** Some times it takes time to flow orders data in to the ADX database. Wait for few minutes and refresh report to see orders data.
```

Comment:
```suggestion
- In the ADX Portal, under Dashboards, open the PoS Orders report and set the time range for "_Last 30 minutes_", and refresh the report to see data.

> **NOTE:** As the Cloud Sync service performs the sync in the backend, it might take a few minutes for orders to show up in ADX.
```